scipy/optimize/_numdiff.py

```diff
@@ -555,8 +558,8 @@ def approx_derivative(fun, x0, method='3-point', rel_step=None, abs_step=None,
         if rel_step is None:
             rel_step = _eps_for_method(x0.dtype, f0.dtype, method)
 
-        J, _nfev = _linear_operator_difference(fun_wrapped, x0,
-                                               f0, rel_step, method)
+        J, _ = _linear_operator_difference(fun_wrapped, x0,
+                                           f0, rel_step, method)
```

Comment:
We don't use `workers` with `as_linear_operator` because:

- the `LinearOperator` may still be used outside the scope in which `workers` is being used.
- there are only two or three function evaluations, which isn't a high priority for being parallelised.